scripts/runtime/VibeRuntime.Common.ps1

```
@@ -261,6 +261,130 @@ function Get-VibeHostContinuationContext {
+        [AllowNull()] [object[]]$RevisionDelta = @()
+    )
+
+    $text = ([string]::Join(' ', @(Get-VibeNormalizedStringList -Values $RevisionDelta))).ToLowerInvariant()
+    if ([string]::IsNullOrWhiteSpace($text)) {
+        return $false
+    }
+
+    return ($text -match 'tdd[^.;。]*not[_ -]?applicable|not[_ -]?applicable[^.;。]*tdd|tdd[^.;。]*(not required|not require|skip|disabled|off)|研究/设计阶段\s*tdd\s*not[_ -]?applicable')
+}
+
+function Split-VibeRequirementRevisionItems {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Split-VibeRequirementRevisionItems' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Split-VibeRequirementRevisionItems'

(PSUseSingularNouns)